bin/pgenv

```diff
@@ -118,15 +135,94 @@ pgenv_check_dependencies(){
     fi
 }
 
+# This function checks for its first argument to be
+# a valid PostgreSQL number, soemthing like 10.5 or 9.5.4.
+# In case no version number is specified, or the
+# version number is not valid (e.g., 10, 0.9) the
+# function aborts the script.
+# If no version at all is specified, the `pgversions` function
+# is invoked to display the user with available versions.
+pgenv_input_version_or_exit() {
+    local version=$1
+
+    if [ -z "$version" ]; then
+        echo "Which version of PostgreSQL do you want?"
+        pgversions
```

Comment:
Should not output from `build`.

Reply:
Yes, even if knowing which versions are already present could help the user choose which one to build. However, fixed.